nixos/modules/services/misc/paperless.nix

```diff
@@ -252,9 +252,60 @@ in
         '';
       };
     };
+
+    exporter = {
+
+      enable = lib.mkEnableOption "regular automatic document exports";
+
+      directory = lib.mkOption {
+        type = lib.types.str;
+        default = cfg.dataDir + "/exports";
+        defaultText = "\${dataDir}/exports";
+        description = "Directory to store exports.";
+      };
+
+      onCalendar = lib.mkOption {
+        type = lib.types.nullOr lib.types.str;
+        default = "01:30:00";
+        description = ''
+          When to run the exporter.
+
+          Fills in the OnCalendar section of a timer and uses a systemd.time(7)
+          format.
+
+          `null` disables the timer; allowing you to trigger the
+          `paperless-exporter` service through other means.
+        '';
+      };
+
+      options = lib.mkOption {
```

Comment:
One last nit: This should probably be called `settings` in accordance with RFC42.

Reply:
Yeah, I've been pondering that already but was divided whether `settings` is right for command line arguments... ;)

I'll go with `settings` as the RFC is referring to CLI arguments, too.